types/asset.ts

```diff
@@ -1,7 +1,7 @@
 import type {
-  LSP4_TOKEN_TYPES,
-  LSP8_TOKEN_ID_FORMAT,
   LinkMetadata,
+  LSP8_TOKEN_ID_FORMAT,
+  LSP4_TOKEN_TYPES,
 } from '@lukso/lsp-smart-contracts'
 
 // TODO use type from lsp package when it's released
```

Comment:
Might be out of scope, but this TODO could be easily addressed I guess? These types are available from the lsp-smart-contracts package

Reply:
I've checked that already but `name` field is still missing in rc.5 apparently.